invokeai/app/api/dependencies.py

```diff
@@ -61,7 +61,7 @@ def initialize(args, config, event_handler_id: int):
         db_location = os.path.join(output_folder, "invokeai.db")
 
         services = InvocationServices(
-            generate=generate,
+            generator_factory=generator_factory,
```

Comment:
We should probably be registering `model_manager` as a service (especially since we'll need to call into it directly from web endpoints).

Reply:
This `generator_factory` slot is defunct, and I've replaced it with:
```
        services = InvocationServices(
            model_manager=get_model_manager(args, config),
            events=events,
            images=images,
            queue=MemoryInvocationQueue(),
            graph_execution_manager=SqliteItemStorage[GraphExecutionState](
                filename=db_location, table_name="graph_executions"
            ),
```

What I've done is to create a `model_manager` slot within the `InvocationServices` object and populate it with the ModelManager object returned by `get_model_manager()`.  The model management initialization routines have been moved into their own `app.services.model_manager_initializer` module, and I've added back support for embeddings and the embedding_path. This was lost when Generate was removed.

Reply:
It looks like maybe the API initialization code wasn't updated like the CLI was? It's still not passing model manager into the services.

Reply:
Have I missed something? I've currently got the following in `ApiDependencies.initialize()`:

```
        services = InvocationServices(
            model_manager=get_model_manager(config),
            events=events,
            images=images,
            queue=MemoryInvocationQueue(),
            graph_execution_manager=SqliteItemStorage[GraphExecutionState](
                filename=db_location, table_name="graph_executions"
            ),
            processor=DefaultInvocationProcessor(),
            restoration=RestorationServices(config),
        )
```